packages/next-on-pages/src/cli.ts

```diff
@@ -4,7 +4,8 @@ import { z } from 'zod';
 import { argumentParser } from 'zodcli';
```

Comment:
Y'all might want to look at [Pastel](https://github.com/vadimdemedes/pastel) now as a replacement for zodcli. It's much better than my thing.

Reply:
Pastel seems to have a bunch of stuff in it... if you're deprecating `zodcli` I think I'd just fallback to something minimalistic/simpler like jargs, etc...

Pastel feels excessive here, don't you think? (unless we were to redo our whole CLI structure)

Reply:
Up to you — I'm just personally thinking of Pastel as the spiritual successor to zodcli :)